components/SearchBar/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { AiOutlineClose } from 'react-icons/ai';
+import { IoIosSearch } from 'react-icons/io';
+import * as styles from './styles';
```

Comment:
check my other comment above -- we should also import each styled component individually. This is for readability purposes as the styles. looks a little messy